content/en/docs/components/spark-operator/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+  - mwielgus
+  - yuchaoran2011
```

Comment:
@ChenYi015 Let's add @andreyvelich to the list of approvers to align with the ownership structure in the [Kubeflow Spark Operator repository](https://github.com/kubeflow/spark-operator/blob/master/OWNERS).

Reply:
We don't need to add myself since I am in the root owner for the website: https://github.com/kubeflow/website/blob/master/OWNERS